psh/tools/psh.py

```diff
@@ -23,45 +23,51 @@
 
 from datetime import datetime, timedelta
 
-EOL = r'(?:\r+)\n'
-EOT = '\x04'
-PROMPT = r'(\r*)\x1b\[0J' + r'\(psh\)% '
+EOL = r"(?:\r+)\n"
+EOT = "\x04"
+PROMPT = r"(\r*)\x1b\[0J" + r"\(psh\)% "
 # according to control sequence introducer pattern
-CONTROL_CODE = r'(\x1b\[[\x30-\x3F]*[\x20-\x2F]*[\x40-\x7E])'
-OPTIONAL_CONTROL_CODE = CONTROL_CODE + r'?'
+CONTROL_CODE = r"(\x1b\[[\x30-\x3F]*[\x20-\x2F]*[\x40-\x7E])"
+OPTIONAL_CONTROL_CODE = CONTROL_CODE + r"?"
 
 
-File = namedtuple('File', ['name', 'owner', 'is_dir', 'datetime', 'datetime_resolution', 'n_links'])
+File = namedtuple("File", ["name", "owner", "is_dir", "datetime", "datetime_resolution", "n_links"])
 
 
 def _readable(exp_regex):
-    ''' Gets more readable expected regex with new lines instead of raw string EOL,
-    and prompt without raw esc sequences. '''
-    exp_regex = exp_regex.replace(EOL, '\n')
-    return exp_regex.replace(PROMPT, '(psh)% ')
+    """Gets more readable expected regex with new lines instead of raw string EOL,
+    and prompt without raw esc sequences."""
+    exp_regex = exp_regex.replace(EOL, "\n")
+    return exp_regex.replace(PROMPT, "(psh)% ")
 
 
 def _check_result(pexpect_proc, result):
-    if result == 'dont-check':
+    if result == "dont-check":
         return
-    elif result == 'success':
+    elif result == "success":
         assert_cmd_successed(pexpect_proc)
-    elif result == 'fail':
+    elif result == "fail":
         assert_cmd_failed(pexpect_proc)
     else:
-        raise Exception(''.join((f'The value {result} for `result` argument is not correct. ',
-                        'Please choose between susccess/fail/dont-check')))
-
-
-def assert_cmd(pexpect_proc, cmd, *, expected='', result='success', msg='', is_regex=False, timeout=-1):
-    ''' Sends specified command and asserts that it's displayed correctly
-    with optional expected output and next prompt. Exit status is asserted depending on `result`'''
+        raise Exception(
+            "".join(
+                (
+                    f"The value {result} for `result` argument is not correct. ",
+                    "Please choose between susccess/fail/dont-check",
+                )
+            )
+        )
```

Comment:
I'm not sure if using this formatting must be applied in every case, I think the previous form was more clear, although this is just my opinion.

Especially since in this case, `join` was used to shorten the lines.

Reply:
Maybe this would be the clearest approach, but again, this is just my opinion.

```suggestion
        raise Exception(
            f"The value {result} for `result` argument is not correct. "
            "Please choose between susccess/fail/dont-check'."
        )
```